src/main/java/me/leoko/advancedban/utils/Punishment.java

```diff
@@ -137,7 +137,7 @@ public void create(boolean silent) {
 
             if (getType().getBasic() == PunishmentType.BAN || getType() == PunishmentType.KICK) {
                 mi.runSync(() -> mi.kickPlayer(getName(), getLayoutBSN()));
-            } else {
+            } else if (getType().getBasic() != PunishmentType.NOTE){
```

Comment:
You just wanted to avoid the message which is good, but you accidentally also disabled the cache update this way

Reply:
Will revise that then. I didn't notice that.